packages/manager/core/generator/app/universes.constant.js

```diff
@@ -0,0 +1,32 @@
+export const UNIVERSES = [
+  'Dedicated',
+  'Focus',
+  'Manager',
+  'Web',
+  'Server',
+  'Hub',
+  'Creation',
+  'BareMetalCloud',
```

Comment:
```suggestion
  'Baremetal',
```

According to the tracking plan on Vrack Services